eng/Versions.props

```diff
@@ -18,18 +18,17 @@
     <MicrosoftNETHostModelVersion>3.0.0-preview5-27615-04</MicrosoftNETHostModelVersion>
     <NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-26011-01</NETStandardLibraryNETFrameworkVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
-    <NuGetBuildTasksPackVersion>5.1.0-rtm.5921</NuGetBuildTasksPackVersion>
+    <NuGetBuildTasksPackVersion>5.2.0-preview2.6020</NuGetBuildTasksPackVersion>
     <NuGetPackagingVersion>$(NuGetBuildTasksPackVersion)</NuGetPackagingVersion>
     <NuGetProjectModelVersion>$(NuGetBuildTasksPackVersion)</NuGetProjectModelVersion>
     <PlatformAbstractionsVersion>2.0.0</PlatformAbstractionsVersion>
     <SystemDiagnosticsFileVersionInfoVersion>4.0.0</SystemDiagnosticsFileVersionInfoVersion>
     <SystemReflectionMetadataVersion>1.5.0</SystemReflectionMetadataVersion>
-    <MicrosoftDotNetSignToolVersion>1.0.0-beta.19218.7</MicrosoftDotNetSignToolVersion>
+    <MicrosoftDotNetSignToolVersion>1.0.0-beta.19279.5</MicrosoftDotNetSignToolVersion>
   </PropertyGroup>
   <!-- Get .NET Framework reference assemblies from NuGet packages -->
   <PropertyGroup>
     <UsingToolNetFrameworkReferenceAssemblies>true</UsingToolNetFrameworkReferenceAssemblies>
-    <MicrosoftNetFrameworkReferenceAssembliesVersion>1.0.0-alpha-5</MicrosoftNetFrameworkReferenceAssembliesVersion>
```

Comment:
@dsplaisted how are we referencing this version now?

Reply:
It's [defined by arcade](https://github.com/dotnet/arcade/blob/ed3510aaa6610514c484e23a4b7f17963bf16e0c/src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props#L83).  I'm not sure why we had it pinned to an earlier version, maybe we were just using the same property name before arcade had this support built in.

Reply:
Cool.